src/main/java/io/botcrafting/botcraft/core/handler/processors/EasterEggListCommand.java

```diff
@@ -0,0 +1,35 @@
+package io.botcrafting.botcraft.core.handler.processors;
+
+import static io.botcrafting.botcraft.configuration.constant.CommandConstant.COMMAND_EASTER_EGG_LIST;
+import static io.botcrafting.botcraft.configuration.constant.MessageConstant.EASTER_EGG_SHOW_COMMANDS;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import io.botcrafting.botcraft.core.handler.chain.MessageChain;
+import io.botcrafting.botcraft.core.model.Message;
+import io.botcrafting.botcraft.core.service.MessageSenderService;
+
+@Component
+public class EasterEggListCommand implements MessageProcessor{
```

Comment:
Command can be confused with another concept in programming, maybe just EasterEggListReplier